javascript/builtins/webassembly/Global.json

```
@@ -51,11 +51,13 @@
               "spec_url": "https://webassembly.github.io/spec/js-api/#dom-global-global",
               "support": {
                 "chrome": {
-                  "version_added": "69"
+                  "version_added": "69",
+                  "notes": "Constructing a Global with a value of \"v128\" produces a TypeError."
```

Let's use code formatting to emphasize that it's a value:

```suggestion
                  "notes": "Constructing a <code>Global</code> with a value of <code>v128</code> produces a TypeError."
```